packages/test/src/test-sinks.ts

```diff
@@ -146,13 +146,17 @@ if (RUN_INTEGRATION_TESTS) {
   });
 
   test('Sink functions are not called during replay if callDuringReplay is unset', async (t) => {
-    const recordedMessages = Array<string>();
+    const recordedMessages = Array<{ message: string; hl: number; isReplaying: boolean }>();
```

Comment:
Make it more readable

```suggestion
    const recordedMessages = Array<{ message: string; historyLength: number; isReplaying: boolean }>();
```